code/game/machinery/porta_turret/portable_turret.dm

```diff
@@ -592,6 +592,7 @@
 	lethal_projectile_sound = 'sound/weapons/lasercannonfire.ogg'
 	desc = "An energy blaster auto-turret."
 
+

```

Comment:
hey what's the point

Reply:
i tried to add a custom turret at one point

Reply:
remove the newline tbh